contrib/scripts/test_localnet_liveness.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+CNT=0
+ITER=$1
+SLEEP=$2
+NUMBLOCKS=$3
+NODEADDR=$4
+
+if [ -z "$1" ]; then
+  echo "Invalid argument: missing number of iterations"
+  echo "sh test_localnet_liveness.sh <iterations> <sleep> <num-blocks> <node-address>"
+  exit 1
+fi
+
+if [ -z "$2" ]; then
+  echo "Invalid argument: missing sleep duration"
+  echo "sh test_localnet_liveness.sh <iterations> <sleep> <num-blocks> <node-address>"
+  exit 1
+fi
+
+if [ -z "$3" ]; then
+  echo "Invalid argument: missing number of blocks"
+  echo "sh test_localnet_liveness.sh <iterations> <sleep> <num-blocks> <node-address>"
+  exit 1
+fi
+
+if [ -z "$4" ]; then
+  echo "Invalid argument: missing node address"
+  echo "sh test_localnet_liveness.sh <iterations> <sleep> <num-blocks> <node-address>"
+  exit 1
+fi
+
+docker_containers=($(docker ps -q -f name=umeed --format='{{.Names}}'))
+
+while [ ${CNT} -lt $ITER ]; do
```

Comment:
is `$ITER` used for only counting a max looping ? if this way, can  we just write `while [ ${CNT} -lt $(expr ${NUMBLOCKS} + 50) ]; do` then we do not need to pass the arg `ITER=$1`

Reply:
We've had this script in the SDK for years at this point. We could probably clean it up. I'm indifferent to it as it's pretty simple.